cli/constants.py

```diff
@@ -6,3 +6,7 @@
 FAILED_EXIT_FROM_COMMAND = -1
 
 NODE_URL_ARGUMENT_HELP_MESSAGE = 'Apply the command to the specified node by its URL.'
+
+CLI_CONFIG_FILE_NAME = 'remme-core-cli'
+CLI_CONFIG_FILE_NAME_EMPTY_FILE = 'remme-core-cli-empty-file'
+CLI_CONFIG_FILE_NAME_WITHOUT_URL = 'remme-core-cli-without-url'
```

Comment:
Move to the tests files. Constants file is a public file for CLI development.

Reply:
@dmytrostriletskyi, done.